src/plugins/console/kibana.json

```diff
@@ -1,12 +1,14 @@
 {
   "id": "console",
-  "version": "kibana",
+  "version": "8.0.0",
+  "kibanaVersion": "kibana",
```

Comment:
I'm curious about this. I see in [the docs](https://github.com/elastic/kibana/blob/master/docs/development/core/server/kibana-plugin-core-server.pluginmanifest.kibanaversion.md) that its the version of kibana the plugin is compatible with, so by specifying "kibana" it means that is compatible with all versions? 🤔

Reply:
I haven't given it too much thoughts to be honest. I just followed what all the other plugins are doing 😄 `"kibana"` seems to be the kibana version declared everywhere.